building/tracks/config-json.md

```diff
@@ -249,7 +249,33 @@ The key features are specified in the top-level `key_features` field which is de
 - `title`: a concise header for the key feature. Its length must be <= 25. Markdown is _not_ supported.
 - `content`: a description of the key feature. Its length must be <= 100. Markdown is _not_ supported.
 - `icon`: the icon to show for the feature. The following icons can be used:
-  - TODO: list icons
+  - `community`
+  - `concurrency`
+  - `cross-platform`
+  - `documentation`
+  - `dynamically-typed`
+  - `easy`
+  - `embeddable`
+  - `expressive`
+  - `extensible`
+  - `fast`
+  - `fun`
+  - `functional`
+  - `garbage`
```

Comment:
```suggestion
  - `garbage-collected`
```

Better? `garbage` alone will just sound like "bad" to many ears.

Reply:
It makes sense to rename the icon to `garbage-collected`. I'll do that.

Reply:
Done: https://github.com/exercism/website-icons/pull/71/files